source/_docs/guides/build-tools/02-create-project.md

```diff
@@ -0,0 +1,99 @@
+---
+title: Build Tools
+subtitle: Create a New Project
+anchorid: create-project
+layout: guide
+buildtools: true
+generator: pagination
+pagination:
+    provider: data.buildtoolspages
+use:
+    - buildtoolspages
+permalink: docs/guides/build-tools/create-project/
+nexturl: guides/build-tools/create-pr/
+previousurl: guides/build-tools/
+editpath: build-tools/02-create-project.md
+---
+In this section we'll use the Terminus Build Tools Plugin to create a new Pantheon Site, a corresponding GitHub repository, and configure CircleCI to run tests.
+
+1. Create a [CircleCI personal API token](https://circleci.com/account/api){.external} and export the value to the `CIRCLE_TOKEN` environment variable (replace `[REDACTED]`):
+
+    <div class="copy-snippet">
+    <button class="btn btn-default btn-clippy" data-clipboard-target="#circle-token">Copy</button>
+    <figure><pre id="circle-token"><code class="command bash" data-lang="bash">export CIRCLE_TOKEN=[REDACTED]</code></pre></figure>
+    </div>
+
+2. Create a [GitHub personal access token](https://help.github.com/articles/creating-an-access-token-for-command-line-use/){.external} and export the value to the `CIRCLE_TOKEN` environment variable (replace `[REDACTED]`):
+
+    <div class="copy-snippet">
+    <button class="btn btn-default btn-clippy" data-clipboard-target="#github-token">Copy</button>
+    <figure><pre id="github-token"><code class="command bash" data-lang="bash">export GITHUB_TOKEN=[REDACTED]</code></pre></figure>
+    </div>
+
+3. Create a new project (replace `pantheon-d8-composer-project` with the name of your new site):
+
+    <div class="copy-snippet">
+      <button class="btn btn-default btn-clippy" data-clipboard-target="#d8-create">Copy</button>
+      <figure><pre id="d8-create"><code class="command bash" data-lang="bash">terminus build:project:create d8 pantheon-d8-composer-project</code></pre></figure>
+    </div>
+
+    <div class="alert alert-info">
+      <h4 class="info">Note</h4>
+      <p markdown="1">Pantheon also maintains Composer based examples for [WordPress](https://github.com/pantheon-systems/example-wordpress-composer){.external} and [Drupal 7](https://github.com/pantheon-systems/example-drops-7-composer){.external} that are currently in alpha, requiring `--stability=alpha` in the command line options. While this guide demonstrates Drupal 8, the same workflow can be achieved on all frameworks.</p>
```

Comment:
We should really tag 1.0.0 stable for WordPress and Drupal 7 before you publish, so that this note is not necessary.